depends/hosts/darwin.mk

```diff
@@ -1,7 +1,7 @@
 OSX_MIN_VERSION=10.12
 OSX_SDK_VERSION=10.14
 OSX_SDK=$(SDK_PATH)/MacOSX$(OSX_SDK_VERSION).sdk
-LD64_VERSION=253.9
+LD64_VERSION=409.12
```

Comment:
All of the LD64_VERSION stuff (used by darwin_CC and darwin_CXX) stuff may be unnecessary now. I believe "-mlinker-version" was a hack around a bug that may have now been fixed.

Could you try removing the ```-mlinker-version=...``` and seeing if the build still succeeds?

Reply:
Have combined your LD64 suggestions into the cctools commit.